src/components/Filters.js

```diff
@@ -2,63 +2,125 @@ import {useFilters } from "../contexts/filter-context/filter-context"
  
 const Filters = () => {
 
-    const {sortBy , rateBy , showFastDeliveryOnly , showCodOnly , dispatch } = useFilters()
+    const {sortBy , rateBy , filterBy , showFastDeliveryOnly , showCodOnly , showCategoryMen , showCategoryWomen , price , dispatch } = useFilters()
 
     return (
         <aside className="filters">
         <div className="filter flex ">
             <p className="filter-heading margin-left margin-bottom margin-top">Filters</p>
             <a className="clear-link margin-right margin-bottom margin-top" onClick = {() => dispatch({type : "CLEAR"})}>Clear</a>
         </div>
+
+    <form>
+        <fieldset id = "range-input">
+        <input
+          id = "price"
+          type="range"
+          list="price-range"
+          min={500}
+          max={4000}
+          step={875}
+          value={price}
+          name = "price"
+          className="price-input"
+          checked = {filterBy && filterBy === "PRICE_FILTER"}
+          onChange={(e) => {dispatch({ type: "PRICE_RANGE_CHANGE", payload1: e.target.value , payload2 : "PRICE_FILTER" })}}
+        />
+        <datalist id="price-range">
+          <option value="500" label="500"></option>
+          <option value="1000"></option>
+          <option value="2000" label="2000"></option>
+          <option value="3000"></option>
+          <option value="4000" label = "4000"></option>
+        </datalist>
+        <div className="flex margin">
+          <span>0</span>
+          <span>1k</span>
+          <span>2k</span>
+          <span>3k</span>
+          <span>4k</span>
+        </div>
+
+        </fieldset>
+        <p className="filter-heading-text margin-left">Rating :</p>
+        <fieldset id = "ratings">
+        <div className="rating margin-top">
+                <input id="rating1" className="margin-left" name="ratings" type="radio" onChange={() =>
+            dispatch({ type: "RATE", payload: "4_STARS_AND_ABOVE" })
+            }
+            checked={rateBy && rateBy === "4_STARS_AND_ABOVE"} />
+                <label for="rating1"> 4 stars and above </label>
+            </div>
+            <div className="rating">
+                <input id="rating2" className="margin-left" name="ratings" type="radio" onChange={() =>
+            dispatch({ type: "RATE", payload: "3_STARS_AND_ABOVE" })
+            }
+            checked={rateBy && rateBy === "3_STARS_AND_ABOVE"}  />
+                <label for="rating2"> 3 stars and above</label>
+            </div>
+            <div className="rating">
+                <input id="rating3" className="margin-left" name="ratings" type="radio" onChange={() =>
```

Comment:
For rating, use array. It would reduce your lines of code too

Reply:
For me , COD available is working fine.
Fixed Add to wishlist from product page.
After adding items to cart, the 'Add to cart' button increases only quantity of the product.
From the wishlist page, move to cart is only moving it to cart but not deleting from wishlist. Check the quantity of cart items after clicking on move to cart button.